src/model.py

```diff
@@ -238,6 +263,21 @@ async def generate(self, request):
                 stream = stream.as_numpy()[0]
             else:
                 stream = False
+            exclude_input_in_output = pb_utils.get_input_tensor_by_name(
+                request, "exclude_input_in_output"
+            )
+            if exclude_input_in_output:
+                exclude_input_in_output = exclude_input_in_output.as_numpy()[0]
+            elif exclude_input_in_output is None and stream:
+                exclude_input_in_output = True
+            else:
+                exclude_input_in_output = False
+
+            self.logger.log_info(
```

Comment:
I dont' see any other params logged per request. Wouldn't it be too verbose?

Reply:
Good point, How about we leave this entry for the case when `stream` is `True`, but `exclude_input_in_output` is intentionally set to `False`. For `stream` case, we never include prompt in this logic, but it would be nice to notify users that `you tried to include prompt, but we will not`

Reply:
updated